src/components/SubCategorySummary/SubCategorySummary.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IndicatorCard from '../Card/IndicatorCard/IndicatorCard';
+import aggregateIndicatorForCities from '../../helpers/aggregateIndicatorForCities';
+import getSubCategorySectionId from '../../helpers/getSubCategorySectionId';
+import { INDICATORS } from '../../constants';
+import style from './SubCategorySummary.scss';
+import COLORS from '../../style/_colors.scss';
+
+const SubCategorySummary = (props) => {
+  const backgroundColor = COLORS[`${props.colorName.toUpperCase()}_${props.colorVariation}`];
```

Comment:
This could *maybe* be a separate function but it isn't used anywhere else and I think would just add complexity.